<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D-mesh generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qng.qrc">
    <normaloff>:/images/app.ico</normaloff>:/images/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_New">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="action_NewCSG"/>
     <addaction name="action_NewSTL"/>
    </widget>
    <addaction name="menu_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="action_SaveMesh"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_SelectAll"/>
   </widget>
   <widget class="QMenu" name="menu_Mesh">
    <property name="title">
     <string>&amp;Mesh</string>
    </property>
    <addaction name="action_Start"/>
    <addaction name="action_Stop"/>
    <addaction name="separator"/>
    <addaction name="action_Refinement"/>
    <addaction name="separator"/>
    <addaction name="action_MeshOptions"/>
   </widget>
   <widget class="QMenu" name="menu_Setup">
    <property name="title">
     <string>&amp;Setup</string>
    </property>
    <addaction name="action_Terminal"/>
    <addaction name="action_ShowModel"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Rotate"/>
    <addaction name="action_Scale"/>
    <addaction name="action_Translate"/>
    <addaction name="separator"/>
    <addaction name="action_Restore"/>
    <addaction name="separator"/>
    <addaction name="action_Parameters"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Mesh"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Setup"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_NewCSG"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="separator"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Start"/>
   <addaction name="action_Stop"/>
   <addaction name="separator"/>
   <addaction name="action_MeshOptions"/>
   <addaction name="separator"/>
   <addaction name="action_Rotate"/>
   <addaction name="action_Scale"/>
   <addaction name="action_Translate"/>
   <addaction name="separator"/>
   <addaction name="action_Parameters"/>
   <addaction name="separator"/>
   <addaction name="action_Terminal"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_NewCSG">
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;CSG</string>
   </property>
   <property name="toolTip">
    <string>Create a new CSG geometry file</string>
   </property>
   <property name="statusTip">
    <string>Create a new NETGEN geometry file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_NewSTL">
   <property name="text">
    <string>&amp;STL</string>
   </property>
   <property name="toolTip">
    <string>Create a new STL file</string>
   </property>
   <property name="statusTip">
    <string>Create a new STL file</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open an existing geometry file</string>
   </property>
   <property name="statusTip">
    <string>Open an existing geometry file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close the current geometry file</string>
   </property>
   <property name="statusTip">
    <string>Close the current geometry file</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the current geometry file</string>
   </property>
   <property name="statusTip">
    <string>Save the current geometry file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="toolTip">
    <string>Save the current geometry file with new name</string>
   </property>
   <property name="statusTip">
    <string>Save the current geometry file with new name</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
   <property name="statusTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo the last action</string>
   </property>
   <property name="statusTip">
    <string>Undo the last action</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo the last undone action</string>
   </property>
   <property name="statusTip">
    <string>Redo the last undone action</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut the current selection's contents to the clipboard</string>
   </property>
   <property name="statusTip">
    <string>Cut the current selection's contents to the clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="toolTip">
    <string>Copy the current selection's contents to the clipboard</string>
   </property>
   <property name="statusTip">
    <string>Copy the current selection's contents to the clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste the clipboard's contents into the current selection</string>
   </property>
   <property name="statusTip">
    <string>Paste the clipboard's contents into the current selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_SelectAll">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="statusTip">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
   <property name="toolTip">
    <string>Start meshing procedure</string>
   </property>
   <property name="statusTip">
    <string>Start meshing procedure</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;top</string>
   </property>
   <property name="toolTip">
    <string>Stop the current process</string>
   </property>
   <property name="statusTip">
    <string>Stop the current process</string>
   </property>
  </action>
  <action name="action_MeshOptions">
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/setup.png</normaloff>:/images/setup.png</iconset>
   </property>
   <property name="text">
    <string>O&amp;ptions...</string>
   </property>
   <property name="toolTip">
    <string>Setup meshing parameters</string>
   </property>
   <property name="statusTip">
    <string>Setup meshing parameters</string>
   </property>
  </action>
  <action name="action_Terminal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/terminal.png</normaloff>:/images/terminal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Terminal</string>
   </property>
   <property name="toolTip">
    <string>Show/hide the Terminal</string>
   </property>
  </action>
  <action name="action_Refinement">
   <property name="text">
    <string>&amp;Refinement</string>
   </property>
   <property name="toolTip">
    <string>Refinement current mesh</string>
   </property>
  </action>
  <action name="action_Rotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/rotate.png</normaloff>:/images/rotate.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate current image</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_Translate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/translate.png</normaloff>:/images/translate.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Translate</string>
   </property>
   <property name="toolTip">
    <string>Translate current image</string>
   </property>
  </action>
  <action name="action_Scale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/scale.png</normaloff>:/images/scale.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Scale</string>
   </property>
   <property name="toolTip">
    <string>Scale current image</string>
   </property>
  </action>
  <action name="action_Restore">
   <property name="text">
    <string>R&amp;estore</string>
   </property>
   <property name="toolTip">
    <string>Restore current image</string>
   </property>
  </action>
  <action name="action_Parameters">
   <property name="icon">
    <iconset resource="qng.qrc">
     <normaloff>:/images/parameters.png</normaloff>:/images/parameters.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Parameters</string>
   </property>
   <property name="toolTip">
    <string>Setup image parameters</string>
   </property>
  </action>
  <action name="action_SaveMesh">
   <property name="text">
    <string>Save &amp;Mesh...</string>
   </property>
   <property name="toolTip">
    <string>Save current mesh</string>
   </property>
  </action>
  <action name="action_ShowModel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Model</string>
   </property>
   <property name="toolTip">
    <string>Show model</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qng.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
